PACKAGE.md

```diff
@@ -50,15 +50,15 @@ osc sr systemsmanagement:SCC:RMT rmt-server openSUSE:Factory
 
 ##### Submit maintenance updates for SLES to the Internal Build Service
 
-###### Get target codesteams where to submit
+###### Get target codestreams where to submit
```

Comment:
`PACKAGING.md` *cough-cough*

Reply:
Which two places?

Reply:
@lagartoflojo https://github.com/SUSE/rmt/blob/master/PACKAGE.md

Reply:
@lagartoflojo arr, nevermind! I thought that was in `README.md` :grimacing: